website/docs/r/distributed_virtual_switch.html.markdown

```diff
@@ -237,7 +237,7 @@ pools, if their use is so desired.
 
 #### Network I/O control traffic classes
 
-There are currently 9 traffic classes that can be used for network I/O
+There are currently 10 traffic classes that can be used for network I/O
```

Comment:
There are 11 as of 7.0.3 - aphapbetically: 

1. Fault Tolerance (FT) Traffic
2. iSCSI Traffic
3. Management Traffic
4. NFS Traffic
5. NVMe over TCP
6. Virtual Machine Traffic
7. vMotion Traffic
8. vSAN Traffic
9. vSphere Backup NFC Traffic
10. vSphere Data Protection Backup Traffic
11. vSphere Replication (VR) Traffic


Fault Tolerance (FT) Traffic
--
iSCSI Traffic
Management Traffic
NFS Traffic
NVMe over TCP
Virtual Machine Traffic
vMotion Traffic
vSAN Traffic
vSphere Backup NFC Traffic
vSphere Data Protection Backup Traffic
vSphere Replication (VR) Traffic



Fault Tolerance (FT) Traffic
--
iSCSI Traffic
Management Traffic
NFS Traffic
NVMe over TCP
Virtual Machine Traffic
vMotion Traffic
vSAN Traffic
vSphere Backup NFC Traffic
vSphere Data Protection Backup Traffic
vSphere Replication (VR) Traffic



Fault Tolerance (FT) Traffic
--
iSCSI Traffic
Management Traffic
NFS Traffic
NVMe over TCP
Virtual Machine Traffic
vMotion Traffic
vSAN Traffic
vSphere Backup NFC Traffic
vSphere Data Protection Backup Traffic
vSphere Replication (VR) Traffic



Fault Tolerance (FT) Traffic
--
iSCSI Traffic
Management Traffic
NFS Traffic
NVMe over TCP
Virtual Machine Traffic
vMotion Traffic
vSAN Traffic
vSphere Backup NFC Traffic
vSphere Data Protection Backup Traffic
vSphere Replication (VR) Traffic

Reply:
Yes there is a new one `nvmetcp` but we can't support it until we bump govmomi. This PR just prevents the provider from crashing

Reply:
Those are in NIOC v3, but oddly, I just noticed that the provider defaults to v2.

Reply:
Gotcha. Approved as-is. We can always resort the list at a later time if you like.